GETTING_STARTED.md

```diff
@@ -1,6 +1,22 @@
 - [Getting Started with the OpenSearch Python Client](#getting-started-with-the-opensearch-python-client)
   - [Setup](#setup)
   - [Sample code](#sample-code)
+    - [Creating a client](#creating-a-client)
+    - [Creating an index](#creating-an-index)
+    - [Adding a document to an index](#adding-a-document-to-an-index)
+    - [Adding documents in bulk](#adding-documents-in-bulk)
+    - [Adding documents in bulk using helper functions](#adding-documents-in-bulk-using-helper-functions)
+    - [Searching for a document](#searching-for-a-document)
+    - [Deleting a document](#deleting-a-document)
+    - [Deleting an index](#deleting-an-index)
+  - [Using plugins](#using-plugins)
+    - [Alerting plugin](#alerting-plugin)
+      - [**Searching for monitors**](#searching-for-monitors)
+      - [**Getting a monitor**](#getting-a-monitor)
+      - [**Creating a monitor**](#creating-a-monitor)
+      - [**Creating a destination**](#creating-a-destination)
+      - [**Getting alerts**](#getting-alerts)
+      - [**Acknowledge alerts**](#acknowledge-alerts)
```

Comment:
Why are these **bold**? :)

Reply:
Everything looked the same on the page. I felt that we would need some separation between the main headings and the sub headings.

Reply:
🤔 that's very un-markdown